### DISCUSSION TITLE
If an item exists on a `many: false` relationship, hide the create button, or rename it to something else

### DISCUSSION THREAD

**Discussion by hypeJunction** [keystonejs/keystone#6787] [Feature requests]
# Bug report

## Describe the bug

A relationship field with `many: false` provides an option to `Create` a new entry, event though one already exists.

![scrnli_8_2_2021_6-16-47 PM](https://user-images.githubusercontent.com/1202761/127892250-9e7e8330-b58f-467f-b182-1e5571fbd707.png)

**Comment by bladey** [comment]
Hey @hypeJunction,

In this case, if you click on `Create Web Meta` it will create another item and set it as the linked item, unlinking the original (if one was set as per your example).

In the `examples-staging/basic` example we have phone numbers, if I set this to `many: false`, and create another phone number it's in place, the item remains but only links the user to the latest one we created.

Example:

<img width="433" alt="image" src="https://user-images.githubusercontent.com/737821/135393446-a7b77fcd-abaa-49e0-94f2-8cbf87e3dde7.png">

<img width="584" alt="image" src="https://user-images.githubusercontent.com/737821/135393535-62ec06c8-807f-4a62-88b7-987be234471b.png">

However I can see the argument that it might not appear that way, and if an item exists on a `many: false` relationship that we hide the create button, or rename it to something else.
